graphql/util.go

```diff
@@ -1290,3 +1290,70 @@ func getAPISubscriptionsForProject(ctx context.Context, projectId string) ([]*re
 	}
 	return res, nil
 }
+
+// GroupProjects takes a list of projects and groups them by their repo. If onlyDefaultedToRepo is true,
+// it only groups projects that are defaulted to the repo
```

Comment:
This comment threw me off slightly—it suggests that projects that don't default to repo are not returned, but they're actually grouped under `""` right? That feels worth mentioning here.

Reply:
good point, I updated the comment.

Reply:
Missing a git push I think :)

Reply:
my bad, pushed, should be there now